pkg/apis/storage/validation/validation.go

```diff
@@ -38,6 +39,17 @@ const (
 	maxVolumeErrorMessageSize     = 1024
 )
 
+// PodInfoOnMountVersions indicates the CSI volume driver
+// requires additional pod information (like podName, podUID, etc.)
+// during mount operations. "v1" is the only supported version now.
+var supportedPodInfoOnMountVersions = sets.NewString(string("v1"))
```

Comment:
Please use the constant

Reply:
Debate: "v1" should be "V1" in bumpy caps beause it is a const, OR "v1" is OK because that's how we spell version numbers in typemeta?

Reply:
Let's go with V1 if we're keeping this (see question about why we need this on types.go, though)

Reply:
Ok